specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/stable/2021-03-30/ScaleUnits.json

```diff
@@ -0,0 +1,258 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-03-30",
+    "title": "Compute Admin Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.Compute.Admin/locations/{location}/scaleUnits/{scaleUnitName}": {
```

Comment:
Microsoft.Fabric.Admin has a resourcetype named scaleunits. It is going to confuse customers. can we name this resourcetype something else?
    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Fabric.Admin/fabricLocations/{location}/scaleUnits/{scaleUnit}": {

Reply:
We have an email thread discussing this with Kirtana Venkatraman and Thomas Roettinger.